fog/api/proto/ingest.proto

```diff
@@ -95,3 +101,48 @@ message SyncKeysFromRemoteRequest {
     /// synced.
     string peer_uri = 1;
 }
+
+message GetIngressKeyRecordsRequest {
```

Comment:
I'm wondering if the filtering is needed - wouldn't it be easier for the client to filter as they please?

Reply:
See comment below.